config/locales/idv/en.yml

```diff
@@ -28,6 +28,7 @@ en:
         dob: Your date of birth must be entered in as mm/dd/yyyy
         personal_key: 'Please enter your personal key for this account. Example: ABC1-DEF2-G3HI-J456'
         ssn: 'Your Social Security Number must be entered in as ###-##-####'
+        state_id_number: Your state id number must be less than 25 characters
```

Comment:
Is the max length 25, or 24? If it's 25, then it needs to be less than 26 characters.

Reply:
The max is 25. Maybe 25 characters or less is what we should be saying here. Message def needs work.